docs/DeveloperGuide.md

```diff
@@ -118,11 +120,22 @@ The `Model`,
 
 ![Structure of the Storage Component](images/StorageClassDiagram.png)
 
-**API** : [`Storage.java`](https://github.com/AY2021S1-CS2103T-F13-2/tp/blob/master/src/main/java/seedu/address/storage/Storage.java)
+**API** : [`Storage.java`](https://github.com/AY2021S1-CS2103T-F13-2/tp/blob/master/src/main/java/seedu/bookmark/storage/Storage.java)
 
 The `Storage` component,
 * can save `UserPref` objects in json format and read it back.
-* can save the address book data in json format and read it back.
+* can save the library data in json format and read it back.
+
+### Algo component
```

Comment:
Would have to take a look at your implementation to confirm, but I don't think that "algorithms" constitute a component on its own. Could they just be part of the Logic component given that the algorithms are used to perform business logic?